app/src/main/java/org/sopt/dosoptjaewon/presentation/main/home/adapter/FriendAdapter.kt

```diff
@@ -0,0 +1,71 @@
+package org.sopt.dosoptjaewon.presentation.main.home.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import coil.transform.CircleCropTransformation
+import com.sopt.common.adapter.ItemDiffCallback
+import org.sopt.dosoptjaewon.databinding.ItemFriendBinding
+import org.sopt.dosoptjaewon.model.Friend
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+class FriendAdapter : ListAdapter<Friend, FriendAdapter.FriendViewHolder>(
+    ItemDiffCallback<Friend>(
+        onItemsTheSame = { old, new -> old.name == new.name },
+        onContentsTheSame = { old, new -> old == new }
+    )
+) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): FriendViewHolder {
+        val binding = ItemFriendBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return FriendViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: FriendViewHolder, position: Int) {
+        val friendInfo = getItem(position)
+        holder.onBind(friendInfo)
+    }
+
+    class FriendViewHolder(private val binding: ItemFriendBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun onBind(friendInfo: Friend) {
+            binding.ivFriendProfile.load(friendInfo.profileImage) {
+                transformations(CircleCropTransformation())
+            }
+            binding.tvFriendName.text = friendInfo.name
+
+            // if today is brithDay
+            if (brithDayConfirm(friendInfo.birthDay)) {
+                with(binding) {
+                    llFriendGift.visibility = ViewGroup.VISIBLE
+                    tvFriendName.text = friendInfo.name + "\uD83C\uDF82"
+                    tvFriendBrithDay.visibility = ViewGroup.VISIBLE
+                    tvFriendBrithDay.text =
+                        friendInfo.birthDay.format(DateTimeFormatter.ofPattern(BIRTHDAY_PATTERN))
+                }
+            } else {
+                // if music is not empty
+                if (friendInfo.music.isNotEmpty()) {
+                    binding.llFriendMusic.visibility = ViewGroup.VISIBLE
+                    binding.tvFriendMusic.text = friendInfo.music
+                }
+            }
+
+            // update is true
+            if (friendInfo.update) binding.vFriendNew.visibility = ViewGroup.VISIBLE
```

Comment:
```suggestion
            binding.vFriendNew.isVisible = friendInfo.update
```
처럼도 쓸 수 있답니다 ~~~~~~~

Reply:
boolean값으로 처리가 가능하군요? 배웠습니다!

Reply:
아 안타깝게도 update 변수가 `boolean` 이여서 적용이 어려울꺼 같아요